src/clr-angular/data/stack-view/stack-block.spec.ts

```diff
@@ -83,6 +86,30 @@ export default function(): void {
       return bFixture.componentInstance.blockInstance;
     }
 
+    describe('Accessibility', () => {
+      let dd: HTMLElement;
```

Comment:
nit: its no longer a `dd` so maybe just use `let label: HTMLElement;` is better.

Reply:
I will fix it with the rebase.